ClientApp/src/app/app.module.ts

```diff
@@ -0,0 +1,54 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
+import { MatButtonModule} from '@angular/material/button';
+import { AppRoutingModule } from './app-routing.module';
+import { MatTabsModule } from '@angular/material/tabs';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatOptionModule } from '@angular/material/core';
+import { MatAutocompleteModule } from '@angular/material/autocomplete';
+import { MatRadioModule } from '@angular/material/radio';
+import { MatDatepickerModule } from '@angular/material/datepicker';
+import { MatNativeDateModule} from '@angular/material/core';
+
+import { AppComponent } from './app.component';
+import { TopBarComponent } from './top-bar/top-bar.component';
+import { MainPageComponent } from './main-page/main-page.component';
+import { CarouselComponent } from './carousel/carousel.component';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```

Comment:
BrowserAnimationsModule in **components** import :(

Reply:
I'm about to block you

Reply:
> BrowserAnimationsModule in **components** import :(

Sasha, I'm about to block you

Reply:
But he's actually right :) Please use logical grouping for your imports.